### ISSUE TITLE
 Failed to load node_modules/svelte-spa-router/Router.svelte

### ISSUE THREAD

**Issue by Perococco** [ItalyPaleAle/svelte-spa-router#181]
Hi,

I'm using the following template:

`npx create-snowpack-app dir-name --template snowpack-svelte-ts-tw`

It works fine until I add the svelte-spa-router dependency like this:

     "dependencies": {
       "svelte": "^3.24.0",
       "svelte-spa-router": "^3.1.0"
     },

Then `npm run build` will give me the following error:

    [snowpack] Failed to load node_modules/svelte-spa-router/Router.svelte
      Try installing rollup-plugin-svelte and adding it to Snowpack (https://www.snowpack.dev/#custom-rollup-plugins)
    [snowpack] Install failed.

This can be reproduced simply by getting the template, adding the dependency and an `import Router from "svelte-spa-router"` in the App.svelte file.

**Comment by ItalyPaleAle** [comment]
I'm not an expert of Snowpack. Have you tried adding `rollup-plugin-svelte` like the error suggests?

**Comment by Perococco** [comment]
Hi,
I tried but the error message is still present. I do not think this is a problem with snowpack (at least directly) because I use svelte-spa-router (great project by the way) with snowpack in another project.

It seems there is an incompatibility with one of the plugin (probably `@snowpack/app-scripts-svelte`) but not sure though.
Anyway, I managed to get what I wanted (tailwind) in my initial project without using the template `snowpack-svelte-ts-tw`.
